index.html

```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *Abhishek Kuamar* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Abhishek Kumar</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank"></a>
+          </p>
+          <p class="about">about you</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://github.com/abhishek44priv" target="_blank" title="First Resource">Github</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Email</a>
```

Comment:
You'll want to make sure to change `<a href="#"` to direct to your second 'Useful Dev Resource', I saw you put 'Email' Maybe you wanted to change that to `<a href="https://mail.google.com"` or something along those lines?